### DISCUSSION TITLE
Create debug command

### DISCUSSION THREAD

**Discussion by Insprill** [Zrips/CMI#4680] [Feature Requests]
**Description of feature/improvement request:**
Create a debug command that puts all info you could need for a bug report into a paste/hastebin then just ask for that link instead of cmi ver, server ver, etc. 
---

### Basic explanation as to why this feature is needed and/or useful to have: ###
It's nice as a developer to have all info you could need about investigating a bug report in a bin vs relying on people to tell you the information. Too many "latest version"'s :P

I have the code for this I use in my plugins and I'd be happy to send it too you😃

**Comment by Zrips** [comment]
In general if people cant manage to provide information from /cmi version output, it will not really change when you have special command for that. But yea, we could add something what would output extra information, if there is any extra we would actually need. /cmi version already includes server version and some extra details about players setup. If you have something extra in mind what we might need to output in this same or special command, i'm all ears.

**Comment by Insprill** [comment]
The way I do it is my command just sends back a hastebin link, and I ask for that. So instead of asking for specific info from the version command, you could just ask for that 1 link with everything you could need in it. [here](https://hastebin.com/lahazewoxi.yaml) is an example of mine. I don't get a ton of info but you get the idea :P
